config.py

```diff
@@ -12,41 +13,74 @@
 
 
 class FilterType(str, Enum):
+    """What to do if the opponent declines our challenge."""
+
     NONE = "none"
+    """Will still challenge the opponent."""
     COARSE = "coarse"
+    """Won't challenge the opponent for some time."""
```

Comment:
Since bools have replaced timers, this should probably read: Won't challenge the opponent again.